python/cuml/test/test_random_forest.py

```diff
@@ -7,30 +22,56 @@
 from sklearn.metrics import accuracy_score
 
 
+def unit_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.unit)
+
+
+def quality_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.quality)
+
+
+def stress_param(*args, **kwargs):
+    return pytest.param(*args, **kwargs, marks=pytest.mark.stress)
+
+
+@pytest.mark.parametrize('nrows', [unit_param(30), quality_param(5000),
+                         stress_param(500000)])
+@pytest.mark.parametrize('ncols', [unit_param(10), quality_param(100),
+                         stress_param(600)])
+@pytest.mark.parametrize('n_info', [unit_param(7), quality_param(50),
+                         stress_param(500)])
 @pytest.mark.parametrize('datatype', [np.float32, np.float64])
 @pytest.mark.parametrize('use_handle', [True, False])
-def test_rf_predict_numpy(datatype, use_handle):
-
-    X, y = make_classification(n_samples=1000, n_features=100,
-                               n_clusters_per_class=1, n_informative=80,
+def test_rf_predict_numpy(datatype, use_handle, n_info, nrows, ncols):
+    train_rows = np.int32(nrows*0.8)
+    X, y = make_classification(n_samples=nrows, n_features=ncols,
+                               n_clusters_per_class=1, n_informative=n_info,
                                random_state=123, n_classes=5)
-    y = y.astype(np.int32)
+    X_test = np.asarray(X[train_rows:, 0:]).astype(datatype)
+    y_test = np.asarray(y[train_rows:, ]).astype(np.int32)
+    X_train = np.asarray(X[0:train_rows, :]).astype(datatype)
+    y_train = np.asarray(y[0:train_rows, ]).astype(np.int32)
+
+    if nrows != 500000:
```

Comment:
Checking against the exact param seems brittle. Should be a `<` if I understand the intent right